docs/core/tools/create-custom-templates.md

```diff
@@ -0,0 +1,232 @@
+| **\<description>** | string | A long description of the package for UI display. |
+| **\<id>**          | string | The case-insensitive package identifier, which must be unique across nuget.org or whatever gallery the package will reside in. IDs may not contain spaces or characters that are not valid for a URL, and generally follow .NET namespace rules. See Choosing a unique package identifier for guidance. |
+| **\<packageType>** | string | Place this element inside a **\<packageTypes>** element among the **\<metadata>** elements. Set the `name` attribute of the **\<packageType>** element to `Template`. |
+| **\<version>**     | string | The version of the package, following the major.minor.patch pattern. Version numbers may include a pre-release suffix as described in [Prerelease Packages](https://docs.microsoft.com/nuget/create-packages/prerelease-packages#semantic-versioning). |
+
+See the [.nuspec reference](https://docs.microsoft.com/nuget/schema/nuspec) for the complete *nuspec* file schema.
+
+**Example**
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>GarciaSoftware.ConsoleTemplate.CSharp</id>
+    <version>1.0.0</version>
+    <description>
+      Creates the Garcia Software console app.
+    </description>
+    <authors>Catalina Garcia</authors>
+    <packageTypes>
+      <packageType name="Template" />
+    </packageTypes>
+  </metadata>
+</package>
+```
+
+[Create the package](https://docs.microsoft.com/nuget/create-packages/creating-a-package#creating-the-package) using the `nuget pack <your_nuspec_file>.nuspec` command.
+
+**Example**
+
+```console
+nuget pack C:\Users\<USER>\Documents\Templates\GarciaSoftware.ConsoleTemplate.CSharp\GarciaSoftware.ConsoleTemplate.CSharp.nuspec
+```
+
+## Installing a template
+
+Install a custom template from a NuGet package on any NuGet feed, by referencing a *nupkg* file directly, or by specifying a file system directory that contains a templating configuration. Use the `-i|--install` option with the `dotnet new` command.
+
+### To install a template from a NuGet package stored at [nuget.org](https://www.nuget.org/):
+
+```console
+dotnet new -i <NUGET_PACKAGE_ID>
+```
+
+**Example**
+
+```console
+dotnet new -i GarciaSoftware.ConsoleTemplate.CSharp
+```
+
+> [!NOTE]
+> The example is for demonstration purposes only. There isn't a `GarciaSoftware.ConsoleTemplate.CSharp` NuGet package at nuget.org. If you run the command, no template is installed. However, you can install a template that hasn't been published to nuget.org by referencing the *nupkg* file directly on your local file system as shown in the next section.
+
+### To install a template from a local *nupkg* file:
+
+```console
+dotnet new -i <PATH_TO_NUPKG_FILE>
+```
+
+**Example**
+
+```console
+dotnet new -i C:/Users/Garcia/GarciaSoftware.ConsoleTemplate.CSharp.1.0.0.nupkg
```

Comment:
This example uses slashes and explicitly names the user (`C:/Users/Garcia`). The example above uses backslahes and user name placeholder (`C:\Users\<USER>`). If you're going to keep these absolute paths, I think they should at least be consistent.